### PR TITLE
Cluster monitoring RN updates

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: http://file.rdu.redhat.com/~choag/041520/rn-4.4-updates-2/release_notes/ocp-4-4-release-notes.html

### CODE REVIEW

**Comment by codyhoag** [comment]
@s-urbaniak this PR adds release notes based on your [tracker comment](https://github.com/openshift/openshift-docs/issues/18426#issuecomment-611559563). Can you review this to make sure I got everything right? Thanks!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -117,6 +117,57 @@ The Dashboards view is now available from the Monitoring section in the web
...
+
+[id="ocp-4-4-cluster-reader-can-read-node-metrics"]
+==== cluster-reader can read node metrics
+
+The `cluster-reader` role can now read node metrics by default.
+
+[id="ocp-4-4-new-cluster-alert"]
+==== Cluster alert for when multiple containers are killed
+
+When multiple containers have been killed within the past 15 minutes, you are
```
Comment (lilic) [inline release_notes/ocp-4-4-release-notes.adoc:134]:
```suggestion
When multiple containers have been killed, due to being of out of memory, within the past 15 minutes, you are
```

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -117,6 +117,57 @@ The Dashboards view is now available from the Monitoring section in the web
...
+
+[id="ocp-4-4-cluster-monitoring-version-updates"]
+==== Cluster monitoring component version updates
+
+The following monitoring components have been upgraded:
+
+* Prometheus: version upgrade from 2.14.0 to 2.15.2
+* Alertmanager: version upgrade from 0.19.0 to 0.20.0
+* Prometheus Operator: version upgrade from 0.34.0 to 0.35.1
+* `kube-state-metrics`: version upgrade from 1.8.0 to 1.9.5
```
Comment (lilic) [inline release_notes/ocp-4-4-release-notes.adoc:168]: Why is kube-state-metrics in code style ticks?

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -231,6 +282,184 @@ You must use the `oc create secret` command instead.
...
+* `admission_quota_controller_unfinished_work_seconds`
+* `non_structural_schema_condition_controller_adds`
+* `APIServiceRegistrationController_unfinished_work_seconds`
+* `admission_quota_controller_work_duration`
+* `autoregister_depth`
+* `autoregister_retries`
+* `kubeproxy_sync_proxy_rules_latency_microseconds`
+* `rest_client_request_latency_seconds`
+* `non_structural_schema_condition_controller_retries`
+
```
Comment (lilic) [inline release_notes/ocp-4-4-release-notes.adoc:462]: We need to also add:
> Drop high granularity request duration buckets - `apiserver_request_duration_seconds_bucket`, leaving enough buckets for meaningful alerting, but drastically reducing cardinality. (38 -> 13 buckets)

**Review by lilic — COMMENTED** [review]
Few comments. Thanks!

File: release_notes/ocp-4-4-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:168]: I'll remove them for consistency with the others. I had seen them with backticks elsewhere in the docs, but it seems the majority are without.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 3 above)
Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:462]: @lilic Are you recommending `apiserver_request_duration_seconds_bucket` be added to the *API server metrics* subsection for removed Kubernetes metrics? Or should this be a separate section (possibly deprecation?) outlining this particular metric and why it should be dropped?

**Comment by s-urbaniak** [comment]
lgtm modulo the comments from @lilic

File: release_notes/ocp-4-4-release-notes.adoc (hunk 3 above)
Comment (lilic) [reply release_notes/ocp-4-4-release-notes.adoc:462]: I guess a separate section makes sense, thanks!

Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:580]: @lilic I added this info via this commit: 6fa3cbd4c313a98d5fa44d76520b17f7b3c592b6. I tried giving a bit more context (found [MON-847](https://issues.redhat.com/browse/MON-847), which is where I dug up some more info), but will definitely need you to review it. Let me know if I'm way off track. Thanks!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -396,6 +625,14 @@ following command will also be deprecated at that time:
 For more information on the upcoming new Operator bundle format and Operator
 Package Manager CLI (`opm`), see the link:https://docs.okd.io/latest/operators/understanding_olm/olm-understanding-olm.html#olm-new-bundle-opm_olm-understanding-olm[upstream OKD documentation].
 
+[id="ocp-4-4-drop-high-granularity-request-duration-buckets"]
+==== Drop high granularity request duration buckets in Prometheus
+
+You should drop high granularity request duration buckets in Prometheus, which
```
Comment (lilic) [inline release_notes/ocp-4-4-release-notes.adoc:631]: Only note is, we do drop them already. So:
```suggestion
We dropped high granularity request duration buckets in Prometheus, which
```

**Review by lilic — APPROVED** [review]
Thanks, one nit, otherwise lgtm, great job! 🎉

File: release_notes/ocp-4-4-release-notes.adoc (hunk 4 above)
Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:631]: ah, got it. Thanks for reviewing @lilic! I'll make that change and get this into the release notes.

**Comment by lamek** [comment]
LGTM.

**Pull request merged**